HOME: /t3home/frejalom
USER: frejalom
SLURM_JOB_ID: 301036
HOSTNAME: t3wn58
CMSSW RELEASE BASE  /cvmfs/cms.cern.ch/slc7_amd64_gcc12/cms/cmssw/CMSSW_14_2_0_pre4
%MSG-i AlpakaService:  (NoModuleName) 09-Apr-2025 07:43:07 CEST pre-events
AlpakaServiceSerialSync succesfully initialised.
Found 1 device:
  - Intel(R) Xeon(R) CPU E5-2698 v3 @ 2.30GHz
%MSG
09-Apr-2025 07:43:26 CEST  Initiating request to open file root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root
%MSG-w XrdAdaptorInternal:  file_open 09-Apr-2025 07:44:33 CEST pre-events
Failed to open file at URL root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root?tried=+1213xrootd-redic.pi.infn.it&xrdcl.requuid=c1c1f4bc-ef3f-4608-8ebe-8bc75106d880.
%MSG
%MSG-w XrdAdaptorInternal:  file_open 09-Apr-2025 07:44:33 CEST pre-events
Failed to open file at URL root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root?tried=+1213xrootd-redic.pi.infn.it,&xrdcl.requuid=c5231118-df88-4270-a674-8fac2156c23d.
%MSG
----- Begin Fatal Exception 09-Apr-2025 07:44:33 CEST-----------------------
An exception of category 'FileOpenError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing input source of type PoolSource
   [2] Calling RootInputFileSequence::initTheFile()
   [3] Calling StorageFactory::open()
   [4] Calling XrdFile::open()
Exception Message:
Failed to open the file 'root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root'
   Additional Info:
      [a] Input file root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root could not be opened.
      [b] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.
' (errno=3011, code=400). No additional data servers were found.
      [c] Last URL tried: root://cms-xrd-global.cern.ch:1094//store/relval/CMSSW_14_2_0_pre4/RelValTTbar_14TeV/GEN-SIM-RECO/PU_141X_mcRun4_realistic_v3_STD_2026D110_PU-v2/2590000/6ce2b869-924c-481a-9970-e34c3d0bea52.root?tried=+1213xrootd-redic.pi.infn.it,&xrdcl.requuid=c5231118-df88-4270-a674-8fac2156c23d
      [d] Problematic data server: cms-xrd-global.cern.ch:1094
      [e] Disabled source: cms-xrd-global.cern.ch:1094
----- End Fatal Exception -------------------------------------------------
logfile
pv_cfg.py
cp: cannot stat 'pv.root': No such file or directory
